x/pool-incentives/keeper/grpc_query.go

```diff
@@ -61,21 +62,25 @@ func (q Querier) GaugeIds(ctx context.Context, req *types.QueryGaugeIdsRequest)
 	return &types.QueryGaugeIdsResponse{GaugeIdsWithDuration: gaugeIdsWithDuration}, nil
 }
 
+// LockableDurations iterates over all gauges, returns distribution info.
```

Comment:
```suggestion
// DistrInfo iterates over all gauges, returns distribution info.
```

Reply:
Err all the comments in this file seem wrong

Reply:
Oh hmm, will check & fix. Sorry about that!

Reply:
Fixed!